tests/lint/pycodestyle/run.sh

```diff
@@ -14,7 +14,7 @@ readonly PYCODESTYLE_CMD="pycodestyle"
 
 # pycodestyle
 error=0
-for file in $(find "${PYTHON_SCRIPT_DIRECTORY}" -name "*.py" | sort); do
+for file in $(find "${PYTHON_SCRIPT_DIRECTORY}" -name "*.py" | grep -v "generator_test_data" | sort); do
```

Comment:
We want to test generated_test_data with multiple lint tools.
Could you consider to remove this limitation?

Reply:
Currently, pylint too does not lint `generator_test_data`.
See config in `.pylintrc`.
```ini
[MASTER]

disable=
    C0111,
    C0302,
    W0511,

variable-rgx=[a-zA-Z_][a-zA-Z0-9_]{0,40}$
argument-rgx=[a-z_][a-z0-9_]{0,40}$
attr-rgx=[a-z_][a-z0-9_]{0,40}$
max-statements=150
max-branches=50
min-public-methods=0
max-public-methods=30
max-locals=30
max-returns=10
max-attributes=10
max-bool-expr=20

ignore-paths=tests/python/fake_bpy_module_test/fake_bpy_module_test/generator_test_data/*
```

Reply:
I tried adding an exclusion to pycodestyle config, but it did not work, so I added it to the linting script.
Also, the tested code is not formatted with Ruff as it is done later at the end.
After the code formatting with Ruff, all pycodestyle errors are fixed, as they are only on spacing and line-length rule.

Reply:
Also, some files in `generator_test_data` have some syntax errors concerning default argument.
```python
def function_1(param_1: int=10, param_2: typing.List['ClassA'], param_3: typing.Optional[float]=4.5) -> bool:
```
`param_2` should have a default argument, otherwise, this is a syntax error.

Reply:
To move forward, either we keep ignoring the `generator_test_data` folder or we add the formatting back into the tests.

Reply:
Thanks. I forgot that there is `ignore-paths` in `.pylintrc`.

You solved this issue by the newest commit, so I will resolve this conversation.